lib/grpc.js

```diff
@@ -269,28 +273,29 @@ module.exports.GoogleProtoFilesRoot = GoogleProtoFilesRoot;
 // Causes the loading of an included proto to check if it is a common
 // proto. If it is a common proto, use the google-proto-files proto.
 GoogleProtoFilesRoot.prototype.resolvePath = function(originPath, includePath) {
+  originPath = path.normalize(originPath);
+  includePath = path.normalize(includePath);
+
   // Fully qualified paths don't need to be resolved.
-  if (includePath.startsWith('/')) {
+  if (path.isAbsolute(includePath)) {
     return includePath;
   }
 
   if (COMMON_PROTO_FILES.indexOf(includePath) > -1) {
     return path.join(googleProtoFilesDir, includePath);
   }
 
-  try {
-    return GoogleProtoFilesRoot._findIncludePath(originPath, includePath);
-  } catch (e) {
-    return protobuf.util.path.resolve.apply(
-        null, [originPath, includePath]);
-  }
+  return GoogleProtoFilesRoot._findIncludePath(originPath, includePath);
```

Comment:
I removed the fallback to protobuf default resolver. I am almost positive that most of the time, if this line fails, the whole thing will not load. Throwing here will give us better errors since the protobuf loadSync method fails silently if a file is not found.